takwimu/templates/takwimu/_includes/dataview/_base.html

```diff
@@ -1,7 +1,9 @@
 {% load wagtailcore_tags %}
 
 <div class="py-2">
-  <h5>{{ indicator.value.title }}</h5>
+  {% if indicator.value.hide_title %}
```

Comment:
Forgot a `not` here i.e. `{%  if not indicator.value.hide_title %}`

Reply:
@kilemensi it should hide the title if `hide_title == True`, so by default `hide_title` is set to False